### PR TITLE
TSK-1814: structured acceptance tests in packages

### PR DESCRIPTION
**Pull request by holgerhagen**
---
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [x] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [ ] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [ ] I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → TSK-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

File: lib/taskana-core/src/test/java/acceptance/task/ServiceLevelPriorityAccTest.java [hunk 1]
```diff
@@ -600,4 +603,38 @@ void should_NotThrowServiceLevelViolation_IfWeekendOrHolidaysBetweenDates() thro
     assertThat(task.getDue()).isEqualTo(getInstant("2020-04-14T07:00:00")); // Tuesday
     assertThat(task.getPlanned()).isEqualTo(getInstant("2020-04-09T07:00:00")); // Thursday
   }
+
+  @WithAccessId(user = "admin")
```
Comment (gitgoodjhe) [inline lib/taskana-core/src/test/java/acceptance/task/ServiceLevelPriorityAccTest.java:607]: Do we need admin for this? As far as i can see there is no role check in that methd (setPlannedPropertyOfTasks)

**Review by gitgoodjhe — CHANGES_REQUESTED** [review]

File: lib/taskana-core/src/test/java/acceptance/task/ServiceLevelPriorityAccTest.java (hunk 1 above)
Comment (holgerhagen) [reply lib/taskana-core/src/test/java/acceptance/task/ServiceLevelPriorityAccTest.java:607]: Actually I didn't change that. However I can image that this is required to make sure that all tasks are updated and not just those, the user has access to. 'taskadmin' might be an alternative user to do that. If this is important to you?

**Comment by mustaphazorgati** [comment]
I suspect a lot of merge conflicts with this PR. 
Please rebase and verify that no new classes have been added by other devs before integrating this.

**Review by gitgoodjhe — APPROVED** [review]
